compiler/rustc_codegen_llvm/src/coverageinfo/mod.rs

```diff
@@ -104,144 +104,66 @@ impl<'tcx> CoverageInfoBuilderMethods<'tcx> for Builder<'_, '_, 'tcx> {
     fn add_coverage(&mut self, instance: Instance<'tcx>, coverage: &Coverage) {
         let bx = self;
 
-        let Coverage { kind, code_region } = coverage.clone();
+        let Some(coverage_context) = bx.coverage_context() else { return };
+        let mut coverage_map = coverage_context.function_coverage_map.borrow_mut();
+        let func_coverage = coverage_map
+            .entry(instance)
+            .or_insert_with(|| FunctionCoverage::new(bx.tcx(), instance));
+
+        let Coverage { kind, code_region: region } = coverage.clone();
```

Comment:
Minor: Any reason why this is `region` rather than `code_region`?

I have a slight preference for sticking with the real field name rather than the inlined parameter name, since it makes this `let` easier to read, and makes the usage of that field a bit easier to track overall.